src/main/java/org/myrobotlab/framework/repo/IvyWrapper.java

```diff
@@ -389,7 +389,12 @@ synchronized public void install(String location, String[] serviceTypes) {
       Set<ServiceDependency> targetLibraries = getUnfulfilledDependencies(serviceTypes);
 
       if (targetLibraries.size() == 0) {
-        info("%s already installed", (Object[]) serviceTypes);
+        StringBuilder sb = new StringBuilder();
+        for (String type : serviceTypes) {

```

Comment:
There might be a merge problem here..  I thought this was fixed in the webcam PR.

Reply:
fixed - merged develop

Reply:
added semi-colon